### ISSUE TITLE
Can it use ResNet12 as backbone?

### ISSUE THREAD

**Issue by WHQ1111** [csyanbin/TPN-pytorch#5]
I try to change the backbone from Conv64-4 to ResNet12, but it is harmful for the accuracy. Is this model not suitable for large networks like ResNet or there is a bug in my code? And I found that the 5-shot situation usually fails, even with Conv64-4.

**Comment by csyanbin** [comment]
Our model is a general framework for ConvNet and ResNet. This repo aims to provide simple code for reproducing the results in our ICLR19 paper. 

For ResNet12, I think you need to carefully tune the optimizer/learning rate (e.g., utilize SGD with momentum, tune lr and decay steps) as well as more iterations. Recent work reproduced our algorithm on ResNet and obtained good performance, e.g., "Learning to Self-Train for Semi-Supervised Few-Shot Classification".
